README.md

```diff
@@ -593,6 +593,7 @@ how types are serialized. Here are the supported annotations:
 | `#[serde(skip_serializing)`                  | Do not serialize this value                                    |
 | `#[serde(skip_serializing_if_empty)`         | Do not serialize this value if `$value.is_empty()` is `true`   |
 | `#[serde(skip_serializing_if_none)`          | Do not serialize this value if `$value.is_none()` is `true`    |
+| `#[serde(disallow_unknown)`                  | Always error during serialization when encountering unknown fields. When absent, unknown fields are ignored for self-describing formats like JSON.  |
```

Comment:
Now that we have container annotations, we should tweak these docs. Can you refactor this section to something like:

```
Structure Annotations:

| Annotation                  | Function                                                                                                                                           |
| ----------                  | --------                                                                                                                                           |
| `#[serde(disallow_unknown)` | Always error during serialization when encountering unknown fields. When absent, unknown fields are ignored for self-describing formats like JSON. |

Field Annotations:

| Annotation                                   | Function                                                       |
| ----------                                   | --------                                                       |
| `#[serde(rename(json="name1", xml="name2"))` | Serialize this field with the given name for the given formats |
| `#[serde(default)`                           | If the value is not specified, use the `Default::default()`    |
| `#[serde(rename="name")`                     | Serialize this field with the given name                       |
| `#[serde(skip_serializing)`                  | Do not serialize this value                                    |
| `#[serde(skip_serializing_if_empty)`         | Do not serialize this value if `$value.is_empty()` is `true`   |
| `#[serde(skip_serializing_if_none)`          | Do not serialize this value if `$value.is_none()` is `true`    |
```